build/tutorials/tutorials-contest/red-dev-avalanche-chainlink-tutorial/ansible-chainlink-avalanche-setup/README.md

```diff
@@ -0,0 +1,19 @@
+# reddev infrastructure automation repository
+Setup requirements:
+1. Ansible v2.9.9 or later
+2. Get Vultr.com API key from https://my.vultr.com/settings/#settingsapi
+3. Create file at ~/.vultr.ini and fill in your API key like so:
+```
```

Comment:
What data type is this block of code?

Reply:
This is not kind of any coding. This is our VPS environment where we installed the ansible v2.9.9 and executed the automated scripts.